packages/rum-core/src/domain/syntheticsContext.spec.ts

```diff
@@ -21,9 +22,16 @@ describe('getSyntheticsContext', () => {
     expect(getSyntheticsContext()).toEqual({
       test_id: 'foo',
       result_id: 'bar',
+      injected: false,
     })
   })
 
+  it('sets the `injected` field to true if the Synthetics test is configured to automatically inject RUM', () => {
+    mockSyntheticsWorkerValues({ publicId: 'foo', resultId: 'bar', injectsRum: true }, 'globals')
+
+    expect(getSyntheticsContext()?.injected).toBeTrue()
```

Comment:
```suggestion
    expect(getSyntheticsContext()!.injected).toBeTrue()
```
no?

Reply:
Both work! But yes, using `!` would raise slightly less confusing error messages if it fails.